webapp/channels/src/components/invitation_modal/invitation_modal.tsx

```diff
@@ -347,7 +347,7 @@ export class InvitationModal extends React.PureComponent<Props, State> {
         if (!emptyInvites) {
             return 'static';
         } else if (this.state.invite.inviteType === InviteType.GUEST) {
-            if (this.state.invite.inviteChannels.channels.length !== 0 ||
+            if (this.state.invite.inviteChannels.channels?.length !== 0 ||
```

Comment:
Nothing against this approach, but should'nt we understand why on channel removing the object is getting removed

Reply:
good point, thanks @M-ZubairAhmed . I checked the flow and it seems to be that somehow the function in the react-select library that sends the channels list accepts the type null as a valid option

<img width="1151" alt="Screenshot 2023-04-27 at 12 29 57" src="https://user-images.githubusercontent.com/10082627/234837174-f8f4d22a-72d9-4f47-9187-c9281c8e3805.png">


and once the value is returned after deleting the elements in the asyncselect component it comes as null.

<img width="457" alt="Screenshot 2023-04-27 at 12 30 47" src="https://user-images.githubusercontent.com/10082627/234837028-636f5421-178b-443c-bbba-5e850fc4d23c.png">

We could add a validation in the onChannelsChange function to guard against null value and in case of null set an empty string as default value if we want to make sure that the channels object is always present.

Reply:
upto you 0/5 on any approach i just wanted to make sure we are not adding guard statements due to errors in object mis matches :)

Reply:
ok, either way this was a quick fix for an annoying error. I will add the guard either way and move this quick fix fast to master.